src/pages/journal/journal.scss

```diff
@@ -1,4 +1,8 @@
 page-journal {
+  .version-number {
+    color: #dbd8d8;
```

Comment:
could we extract this into the colours array and also update the journal.scss file to use the colour array for this colour as well

Reply:
sure